esp8266-fastled-webserver/PridePlayground.cpp

```diff
@@ -1,3 +1,5 @@
+#include "common.h"
```

Comment:
SIMPLE FILE REVIEW

File existed as PridePlayground.h in many of the active branches.

The main change was to use a common base function for both
the fibonacci and non-fibonacci variants.